<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabForm</class>
 <widget class="QWidget" name="TabForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelQuantityTitle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>8</y>
     <width>50</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxQuantityValue">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>28</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelPriceValue">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>31</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0 ₽</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxChoice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>230</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelPriceTitle">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>8</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditSpecifications">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>344</width>
     <height>56</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
